ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -9200,9 +9200,9 @@ static int zend_jit_do_fcall(dasm_State **Dst, const zend_op *opline, const zend
 			func = call_info->callee_func;
 		}
 		if ((op_array->fn_flags & ZEND_ACC_TRAIT_CLONE)
-		 && JIT_G(current_frame)
-		 && JIT_G(current_frame)->call
-		 && !JIT_G(current_frame)->call->func) {
+		 && (!JIT_G(current_frame) ||
+		     !JIT_G(current_frame)->call ||
+		     !JIT_G(current_frame)->call->func)) {
```

Comment:
Nit:

```suggestion
		 && (!JIT_G(current_frame)
		  || !JIT_G(current_frame)->call
		  || !JIT_G(current_frame)->call->func)) {
```

Would be the consistent formatting.

Reply:
I tried this when I originally wrote the patch, but this style doesn't work because `||` has a special meaning in dynasm.

Reply:
Ahh, thank for the explanation.